### PR TITLE
WRQ-27717: Updated Cordova template to use sandstone

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated Cordova template to use sandstone

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated Cordova template to use sandstone

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Updated Cordova template dependencies

### Links
[//]: # (Related issues, references)
WRQ-27717

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: packages/cordova/template/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "private": true,
   "repository": "",
   "enact": {
-    "theme": "moonstone",
+    "theme": "sandstone",
```
Comment (daniel-stoian-lgp) [inline packages/cordova/template/package.json:22]: can you also update the "version" mentioned above?
Let's put version 2 and then see other opinions

File: packages/cordova/template/src/App/App.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import kind from '@enact/core/kind';
-import MoonstoneDecorator from '@enact/moonstone/MoonstoneDecorator';
-import Panels from '@enact/moonstone/Panels';
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
+import Panels from '@enact/sandstone/Panels';
```
Comment (daniel-stoian-lgp) [inline packages/cordova/template/src/App/App.js]: sort imports

File: packages/cordova/template/src/views/MainPanel.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import Button from '@enact/moonstone/Button';
+import Button from '@enact/sandstone/Button';
 import kind from '@enact/core/kind';
-import {Panel, Header} from '@enact/moonstone/Panels';
+import {Panel, Header} from '@enact/sandstone/Panels';
```
Comment (daniel-stoian-lgp) [inline packages/cordova/template/src/views/MainPanel.js]: sort imports

**Review by daniel-stoian-lgp — APPROVED** [review]

File: packages/cordova/template/package.json (hunk 1 above)
Comment (stanca-pop-lgp) [reply packages/cordova/template/package.json:22]: I believe version number is for the user when starting a new project using this template, so it should be 1.0.0.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**